upgraderesponder/service.go

```diff
@@ -189,6 +193,91 @@ func (s *Server) createDB(name string) error {
 	return nil
 }
 
+func (s *Server) includeTags() map[string]bool {
```

Comment:
Sorry for the confusion. This repo is consumed by other projects to. So a white list tag should be configable via configmap similar to a resp.json file. I can take care of this one in #47